benchopt/benchmark.py

```diff
@@ -277,6 +277,20 @@ def validate_solver_patterns(self, solver_patterns):
 
         _validate_patterns(all_solvers, solver_patterns, name_type='solver')
 
+    def validate_objective_filters(self, objective_params):
+        "Check that all objective filters match at least one objective setup"
+
+        # List all choices of objective parameters
+        all_objectives = _list_all_parametrized_names(
+            self.get_benchmark_objective()
+        )
+
+        _validate_patterns(
+            all_objectives,
+            objective_params,
+            name_type="objective"
+        )
```

Comment:
Doesn't it pass on a single line?

Reply:
No, it is the reason why I changed the name `all_objectives` by something shorter in the previous commit